tests/pfcwd/test_pfcwd_function.py

```diff
@@ -732,6 +732,73 @@ def storm_detect_path(self, dut, port, action):
         self.traffic_inst.verify_wd_func(action, self.rx_action, self.tx_action)
         return loganalyzer
 
+    def storm_detect_path_cisco_8000(self, dut, port, action):
```

Comment:
Don't need to check twice, just send some background traffic and check if pfcwd can trigger normally.

Reply:
The problem is that the new pfcwd behaviour is *not* to trigger watchdog, so I expressly put a check for that.

Reply:
but it can be triggered with some background traffic right?

Reply:
@kevinskwang , same response as above: As we need to cover both cases (without and with traffic), background traffic is not needed here. We verify the WD behaviour first without traffic(not triggered), and then with traffic.